Import the service rules (#5)
DFE-Digital/ecf2: documentation/service-rules/dfe-users.md

# Department for Education users

> The goal for ECF2 is that the application will be self-sustainable. That means

Reviewer: ```suggestion
The goal for ECF 2 is that the application will be self-sustainable. That means
```

developers shouldn't be needed in the everyday operation of the application.

This list contains the responsibilities developers have now, but should be

> made available via the admin interface in ECF2:

Reviewer: ```suggestion
made available via the admin interface in ECF 2:
```

- The DfE has previously advised of the possibility that participants
  may be registered as duplicates with multiple `participant_ids`. Where
  the DfE identifies duplicates, it will fix the error by 'retiring'
  one of the participant IDs, then associating all records and data
  under the remaining ID.
- Manual process for merging/closing/opening schools and greatly

>   depends on things like whether the school has a successor,

Reviewer: I'm not sure this would fix the sentence.

  selected
- If the ECT doesn't have an induction start date recorded, they are
  assigned a temporary cohort to enable access training while the AB
  has not yet recorded their induction start date
- Replacement mentors
- Cohort allocation
- [Eligibility for funding](https://teacher-cpd.design-history.education.gov.uk/manage-training/iterations-to-eligibility-checks/)
- Syncing our school list with GIAS
- Sending out reminder emails -- manual/scheduled comms
- Checking eligibility
- Setting up cohorts & schedules
- Assigning participants to a cohort temporarily/permanently
- Setting/updating participant statuses
- Calculating grant funding

## Key

| Emoji | Meaning                       |
| ----- | -------                       |
| 🙋    | user need                     |
| 📜    | policy (explicit or intended) |
| 📚    | contracts & funding           |
| 💻    | digital service               |
| 📊    | data                          |
| 🔒    | security / GDPR               |

## Calculating payments for providers

What is referred to as the 'payment engine' is a piece of code in the
same ECF application that calculates payments for LPs.

> Provider payments are made up of:

Reviewer: Are you happy to add the link to the relevant bit? 

Markdown explanation here: https://commonmark.org/help/

> Provider payments are made up of:

Reviewer 2: ```suggestion
[Provider payments are made up of](https://educationgovuk.sharepoint.com/:w:/s/TeacherServices/ERtdlODXiLBFlEVBMLPSL6cBJno1HDIK-Zfdn9bS1ZuE-g?e=ORWQ3p):
```

Author: Added in [951f946](https://github.com/DFE-Digital/ecf2/pull/5/commits/951f946966a65a867c380bcaacbd4b2c7bfc6e64)

### Uplift fees

The uplift fee is a bonus that is paid to providers for recruiting
participants from schools in deprived areas.

-   It is £100 per eligible participant that receives a Started
    declaration.
-   It is paid as a single payment at the first milestone.
-   Eligibility is determined by the participant's school's score for
    sparsity and pupil premium (updated before opening registrations
    each year).
-   Although a participant may be eligible for both pupil premium (£100)
    and sparsity uplift (£100) payments, they will only receive a £100
    payment in total.

### Output fees

The output payment is a variable amount paid according to schedule
milestone dates. There are typically 6 output payments (according to a
standard schedule).

- It is based on the number of declared participants, not the
  recruitment target.
- It is calculated as 60% of the banded participant price, multiplied
  by the participants per band, multiplied by the milestone weighting.
- Started and completed milestones are weighted at 20%, the 4 retained
  milestones at 15% each.

> <!-- questions copied from original doc:
> 
> How do 'Extended' declarations work??? How do replacement mentors fit
> in?
> 
> Ask from contract managers to add 3 additional declarations for ppts on
> extended schedules -- on top of the standard 6
> -->

Reviewer: ```suggestion
```

If these validation checks are passed, the provider's contract for the
academic year ('cohort' / funding pot) is referenced to check:

- The value of the declaration for the participant
- Whether the participant has any additional associated uplifts
- The banding the participant should be assigned to
- Whether the count of declarations is below the LP's recruitment
  target, i.e. below the maximum number of declarations allowed for
  the contract

Based on these details, the output fee amount is calculated and added to
the next available statement.

### Manual adjustments

There are some one-off additional payments added to financial statements
(e.g. IT service charges). These are added manually by contract
managers, outside of the calculation. Contract managers enter this in a
field in the finance tool.

### Clawbacks

DfE can claim back money from providers via clawbacks. If money was paid
in error and declarations are voided, rather than DfE expecting LPs to
pay the money back in a separate process - we deduct the amount due from
the amount to be paid on the financial statement. Clawbacks cover the
different declaration types (e.g. started, retained 1 etc) and uplift.

> <!-- more notes from original doc:
> 
> Need details - speak to Colin
> 
> Usually provider submits in error
> 
> Participants moving cohorts -- need to void declarations first. In this case
> provider would have to resubmit the declarations that have been
> voided
> -->

Reviewer: ```suggestion
```
